x/tokenomics/types/params.go

```diff
@@ -201,3 +223,32 @@ func ValidateMintAllocationSum(params *Params) error {
 
 	return nil
 }
+
+// ValidateDaoRewardAddress validates the DaoRewardAddress param.
+func ValidateDaoRewardAddress(daoRewardAddress any) error {
+	daoRewardAddressStr, ok := daoRewardAddress.(string)
+	if !ok {
+		return ErrTokenomicsParamInvalid.Wrapf("invalid parameter type: %T", daoRewardAddress)
+	}
+
+	// DEV_NOTE: Empty string is valid if mint_allocation_dao is 0.
+	if daoRewardAddressStr == "" {
+		return nil
+	}
+
+	if _, err := sdk.AccAddressFromBech32(daoRewardAddressStr); err != nil {
+		return ErrTokenomicsParamInvalid.Wrapf("invalid dao reward address %q: %s", daoRewardAddressStr, err)
+	}
+
+	return nil
+}
+
+// ValidateDaoRewardParams validates that the dao_reward_address param is not an
+// empty string so long as the mint_allocation_dao param is not 0.
```

Comment:
`// empty string so long as the mint_allocation_dao param is not 0.`

I understand WHY you did this, but I'd rather keep it simpler. Less prone to bugs.